integration/spark/shared/src/main/java/io/openlineage/spark/api/InternalConvertedQueryPlanVisitor.java

```diff
@@ -0,0 +1,20 @@
+/*
+/* Copyright 2018-2024 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+package io.openlineage.spark.api;
+
+import io.openlineage.client.OpenLineage;
+import lombok.NonNull;
+import org.apache.spark.sql.catalyst.plans.logical.LogicalPlan;
+
+public abstract class InternalConvertedQueryPlanVisitor<
```

Comment:
why do we need a new class for this? could we add `getInternalName` method to QueryPlanVisitor?